tests/module_loading/module/module.c

```diff
@@ -37,3 +39,21 @@ int add_with_constant(volatile int a, volatile int b)
         t = -1;
     return t;
 }
+
+#define TEST_SYMBOL(name)        \
+    do                           \
+    {                            \
+        void* ptr = (void*)name; \
```

Comment:
Does this need to be volatile so that the compiler doesn't optimize away the assignment?

Reply:
done